drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -45,6 +45,10 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   scope :in_progress, -> { where(project_id: nil) }
 
+  def project
+    super || Hmis::Hud::Project.find(wip.project_id)
+  end
+
```

Comment:
Is this a bad idea? I think it will make the code cleaner so we don't always need to worry about whether an Enrollment we're working with is in-progress or not. But, it does lead to an unnecessary database lookup on line 76.

Reply:
I _think_ it's okay, but I defer to @eanders for the final word

Reply:
Yeah, that looks ok.  Alternatively, you could do something like.
```suggestion
  def fetch_project
    project || Hmis::Hud::Project.find(wip.project_id)
  end
```

But I'm not sure it's any better.

Reply:
I guess it will depend what happens if you call `enrollments.join(:project)`  Hopefully it does the right thing and ignores this completely.

Reply:
Yep, join ignores this.

Reply:
I worry about using a different name like `fetch_project` because we will inevitably use `project` instead of `fetch_project` somewhere, and introduce bugs that make operations on "WIP enrollments" break. I can't think of any case where we would _want_ `enrollment.project` to return `nil` for wip enrollmments.

Reply:
Yep, I'm good with this as written.